### PR TITLE
Easier on chain default setup

### PR DESCRIPTION
**Pull request by yondonfu**
This PR includes a few minor changes that should make it easier for an on-chain default setup to work right after deployment.

- 8d5fd71: Increase the default ETH funding for accounts and also increase the default deposit and reserve for broadcasters. This should ensure that deposit/reserve funds are not an issue when trying to test an on-chain payment workflow immediately after deployment. Also removed some commented out code that didn't seem needed anymore
- 440e7af: We need to specify `pricePerUnit` and `pricePerPixel` in the standard setup when activating an orchestrator due to livepeer/go-livepeer#993
- c62b57a: Update some flags and options (seems that using the `livepeerBinaryPath` option is no longer recommended and building a new image using `make localdocker` within the `go-livepeer` directory seems easier?) in the default local config (`examples/local.js`)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -478,13 +479,10 @@ Plese note now GCP logging is truned off by default and should be turned on expl
...
       }
       console.log('Depositing....')
-      // await this.api.fundDepositAndReserve(['all'], '5000000000', '500000000000000000')
-      // await this.api.fundDepositAndReserve(['orchestrators'], '1', '2')
-      // await this.api.fundDeposit(['broadcasters'], '1')
       console.log('Initialize round...', onames)
       await this.api.initializeRound([`${onames[0]}`])
       await wait(2000)
-      await this.api.fundDepositAndReserve(['broadcasters'], '2500000000', '1500000000')
+      await this.api.fundDepositAndReserve(['broadcasters'], '1000000000000000000000', '1000000000000000000000')
```
Comment (kyriediculous) [inline src/index.js:485]: What do you think about moving these concerns into the passed in `config`? Same with the amount passed to `fundAccount()`? Not sure if it adds value as hardcoding fairly large numbers does suffice for our purposes.

Comment (yondonfu) [reply src/index.js:485]: You mean using the `Api` object provided by `src/api` to create a script that is executed in the callback for `th.run()` (similar to what is done in `examples/10broadcasters.js`)? We could do that, but if we don't change the standard setup behavior then any test-harness config file that specifies `standardSetup = true` would not allow for a payment workflow that works right out of the box once the nodes are spun up. I think its useful for the standard setup to allow everything to "just work" and then if we want to test specific non-standard behavior we can go into the config file and customize how the nodes are setup.

Comment (kyriediculous) [reply src/index.js:485]: > I think its useful for the standard setup to allow everything to "just work" 

That's true, thanks for clarifying !

**Review by kyriediculous — APPROVED** [review]
LGTM, Might want to wait for Yahya to approve as well although I use this exact same setup locally just fine. 🚀

**Review by ya7ya — APPROVED** [review]
this looks good, Thanks for the smoother setup :+1:

**Pull request merged**